### DISCUSSION TITLE
Quarkus + Cassandra

### DISCUSSION THREAD

**Discussion by jaiprasad266** [quarkusio/quarkus#25284] [Q&A] [answered]
Hi , 

Followed the below doc on to build the sample quarkus cassandra application.
https://quarkus.io/guides/cassandra

Facing build errors , have included the errors.

dependency version used:

<cassandra-quarkus.version>1.1.2</cassandra-quarkus.version>
<driver.version>4.14.1</driver.version>

<dependency>
      <groupId>com.datastax.oss.quarkus</groupId>
      <artifactId>cassandra-quarkus-client</artifactId>
      <version>${cassandra-quarkus.version}</version>
    </dependency>

    <dependency>
      <groupId>com.datastax.oss</groupId>
      <artifactId>java-driver-core</artifactId>
      <version>${driver.version}</version>
    </dependency>

    <dependency>
      <groupId>com.datastax.oss</groupId>
      <artifactId>java-driver-query-builder</artifactId>
      <version>${driver.version}</version>
    </dependency>

    <dependency>
      <groupId>com.datastax.oss</groupId>
      <artifactId>java-driver-mapper-runtime</artifactId>
      <version>${driver.version}</version>
    </dependency>

Caused by: javax.enterprise.inject.UnsatisfiedResolutionException: Unsatisfied dependency for type org.acme.main.FruitDao and qualifiers [@Default]
	- java member: org.acme.main.FruitService#dao
	- declared on CLASS bean [types=[org.acme.main.FruitService, java.lang.Object], qualifiers=[@Default, @Any], target=org.acme.main.FruitService]
	at io.quarkus.arc.processor.Beans.resolveInjectionPoint(Beans.java:428)
	at io.quarkus.arc.processor.BeanInfo.init(BeanInfo.java:524)
	at io.quarkus.arc.processor.BeanDeployment.init(BeanDeployment.java:260)
	... 13 more

I have the FruitDao class annotated with @Dao and the FruitMapper class with annotations @Mapper and @Daafactory just like the documentation .

Any inputs will be of help.

**Comment by jaiprasad266** [comment, marked as answer]
pom was missing the cassandra-quarkus-mapper-processor plugin . Build is successful.

**Comment by TonyCheater** [comment]
i have same error and nothing helps, seems instances of Dao and DaoMapper are not created by libs....

**Comment by TonyCheater** [comment]
Found solution)
add
```
    compileOnly 'com.datastax.oss.quarkus:cassandra-quarkus-quickstart:1.1.2'
    annotationProcessor 'com.datastax.oss.quarkus:cassandra-quarkus-mapper-processor:1.1.2'
```
